packages/xsnap/src/xsnap.js

```diff
@@ -106,9 +120,16 @@ export function xsnap(options) {
   let baton = Promise.resolve();
 
   /**
-   * @returns {Promise<Uint8Array>}
+   * @typedef {Object} RunResult
```

Comment:
`Object` is more or less `any`, yes?

How about:

```
@typedef {{ reply: Uint8Array, meterStats: unknown }} RunResult
```

Reply:
> `Object` is more or less `any`, yes?

Not when it has any `@property` tags underneath it.